### PR TITLE
added method PaymentRequest::setCustomParam($field,$value)

### PR DESCRIPTION
**Pull request by Vitexus**
Hallo i just added method needed to set 'DESCRIPTION' field
++ PHP Doc comments

Please merge if you agree with my solution

Thank you for Your Time

### CODE REVIEW

**Comment by newPOPE** [comment]
Hello, thank for PR. I disagree with general name of method custom param. Please use more precisely name (e.g. `setDescription()`). And why is php doc comments important?

**Comment by Vitexus** [comment]
Important not - but i like feed my typehinter to help me write code faster. 
I can send pull request only with it.

**Comment by newPOPE** [comment]
- php doc - understood (you can keep it)
- method add - rename it only for `description`

and PR will be merged :)

File: src/AdamStipak/Webpay/PaymentRequest.php [hunk 1]
```diff
@@ -72,12 +72,11 @@ public function setMerchantNumber ($number) {
...
-   * Add Custom parameter to request fields
+   * Add Description parameter to request fields
    * 
-   * @param string $field eg DESCRIPTION
-   * @param mixed  $value field value
+   * @param string  $value field value
    */
-  public function setCustomParam($field,$value){
+  public function setDescription($value){
     $this->params[$field] = $value;
```
Comment (newPOPE) [inline src/AdamStipak/Webpay/PaymentRequest.php]: should be `$this->params['description']` ?

**Comment by Vitexus** [comment]
Now.

File: src/AdamStipak/Webpay/PaymentRequest.php (hunk 1 above)
Comment (Vitexus) [reply src/AdamStipak/Webpay/PaymentRequest.php]: Sorry. Fixed

**Review by Vitexus — COMMENTED** [review]
My mistake. Fixed

**Pull request merged**